wire-ios/Wire-iOS Share Extension/PasscodeTextField.swift

```diff
@@ -45,9 +45,12 @@ class PasscodeTextField: AccessoryTextField {
 
         iconButton.tintColor = UIColor.Team.textColor
         iconButton.setBackgroundImage(UIImage.singlePixelImage(with: .clear), for: state)
-
-        iconButton.adjustsImageWhenDisabled = false
-
+        iconButton.configurationUpdateHandler = { button in
+            switch button.state {
+            default:
+                button.imageView?.tintAdjustmentMode = .normal
```

Comment:
Why did you implement these changes only for `disabled` state in the `ValidatedTextField.swift` ? Shouldn't be the same?